gatsby-config.js

```diff
@@ -133,7 +133,7 @@ module.exports = {
                   url: nodeUrl,
                   guid: nodeUrl,
                   custom_elements: [
-                    {"dc:creator": frontmatter.author.name },
+                    {"dc:creator": frontmatter.authors.name }, //changed
```

Comment:
This will not fail on develop, but will fail on build. We need to make sure to loop back to fix issues with RSS and ensure it handles single and multi-authors properly. We do currently have users that use our RSS feed so we should keep consistent behavior